packages/paste-core/components/tabs/src/TabPanels.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {isPaddingTokenProp, PaddingProps} from '@twilio-paste/style-props';
+
+export interface TabPanelsProps extends PaddingProps {
+  children: React.ReactNode;
+}
+
+const TabPanels = React.forwardRef<HTMLDivElement, TabPanelsProps>(
+  ({children, padding, paddingTop, paddingBottom, paddingRight, paddingLeft, ...props}, ref) => {
```

Comment:
Curious why you choose to expose padding on the panels, instead of the panel?

For the Alpha would it be long term safer to not expose these just yet, _just_ incase we actually don't want folks doing this when we do a full design round?

Reply:
I put it on the wrapper because I figured we should enforce that each panel has consistent padding, whatever that may be. But I agree, I'll disable it for now.